geomesa-nifi-processors/geomesa-nifi-processors-accumulo/src/test/scala/org/geomesa/nifi/processors/accumulo/PutGeoMesaAccumuloTest.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.geomesa.nifi.processors.accumulo
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.nifi.util.TestRunners
+import org.geomesa.nifi.processors.AbstractGeoIngestProcessor
+import org.geomesa.nifi.processors.AbstractGeoIngestProcessor.IngestMode
+import org.geomesa.nifi.processors.datastore.PutGeoMesaAccumulo
+import org.geotools.data.DataStoreFinder
+import org.junit.{Assert, Test}
+import org.locationtech.geomesa.accumulo.MiniCluster
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreParams
+import org.locationtech.geomesa.utils.collection.SelfClosingIterator
+
+class PutGeoMesaAccumuloTest extends LazyLogging {
```

Comment:
First, this unit test is amazing.  One of the most useful things I've read recently!